frameworks/cassandra/tests/conftest.py

```diff
@@ -0,0 +1,37 @@
+import os
+import json
+import pytest
+import shakedown
+
+
+@pytest.fixture(scope='session', autouse=True)
+def configure_universe(request):
+    stub_urls = {}
+
+    # prepare needed universe repositories
+    stub_universe_urls = os.environ.get('STUB_UNIVERSE_URL')
+    if not stub_universe_urls:
+        return
+    for url in stub_universe_urls.split(' '):
+        package_name = url.split('/')[4] # TODO: does this need to be a random string?
```

Comment:
Yes. It currently uses a random string. From the build logs:
https://teamcity.mesosphere.io/viewLog.html?tab=buildLog&logTab=tree&filter=debug&expand=all&buildId=694157&_focus=988
```
[06:20:38][Step 1/1] Adding repository: testpkg-5iyxntqs https://infinity-artifacts.s3.amazonaws.com/autodelete7d/cassandra/20170627-062026-8JIXD6Ekopw6LSMu/stub-universe-cassandra.json
```